G4SOLAr/extern/CMakeLists.txt

```diff
@@ -58,3 +58,15 @@ ExternalProject_Add("marley"
   INSTALL_COMMAND cmake -E echo "MARLEY: Skipping install step"
   )
 
+ExternalProject_add("ecosmics"
+  SOURCE_DIR ${PROJECT_SOURCE_DIR}/src/ecosmics
```

Comment:
Alternatively, you can change these suffixes here

Reply:
Should all be fixed now!